.changeset/cold-experts-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/utils": minor
```

Comment:
todo: could you set a patch instead of minor, we reserve the minor bump for more involved breaking changes

Reply:
You're absolutely right!

sorry, my first time using changesets. Thanks for the correction!